Conductor/Client/ApiClient.cs

```diff
@@ -160,20 +161,47 @@ private byte[] GetBytes(Stream input)
         /// <param name="contentType">Content Type of the request</param>
         /// <returns>Object</returns>
         public Object CallApi(
-            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
+            String path, Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
             Dictionary<String, String> headerParams, Dictionary<String, String> formParams,
             Dictionary<String, FileParameter> fileParams, Dictionary<String, String> pathParams,
-            String contentType)
+            String contentType, Configuration configuration = null)
         {
-            var request = PrepareRequest(
-                path, method, queryParams, postBody, headerParams, formParams, fileParams,
-                pathParams, contentType);
+            RestResponse response = null;
+            bool isRetried = false;
+            bool isTokenRefreshed = false;
```

Comment:
We need only one variable to check the token is refreshed. i.e. isTokenRefreshed

Reply:
@manan164  This has been taken care.